doc/fwutil/fwutil.md

```diff
@@ -405,6 +459,33 @@ _sonic-buildimage/device/<platform_name>/<onie_platform>/platform_components.jso
 }
 ```
 
+3. Non modular chassis platform with Boot action
+```json
+{
+    "chassis": {
+        "Chassis1": {
+            "component": {
+                "BIOS": {
+                    "firmware": "/usr/share/sonic/<platform_name>/<onie_platform>/fw_update/bios.bin",
+                    "version": "0ACLH003_02.02.010",
+                    "boot": "cold"
+                },
+                "CPLD": {
+                    "firmware": "/usr/share/sonic/<platform_name>/<onie_platform>/fw_update/cpld.bin",
+                    "version": "10",
+                    "boot": "powercycle"
+                },
+                "FPGA": {
+                    "firmware": "/usr/share/sonic/<platform_name>/<onie_platform>/fw_update/fpga.bin",
+                    "version": "5",
+                    "boot": "cold"
```

Comment:
Can this boot option be removed from this json file since the platform utility will handle the boot-type passed by the user as explained below.